workflow/notebooks/h12-signal-detection.ipynb

```diff
@@ -184,30 +184,6 @@
     "phasing_analysis"
    ]
   },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "id": "3dd737c8-96c9-4659-8ce0-aa1e8212d5cd",
-   "metadata": {
-    "tags": [],
-    "vscode": {
-     "languageId": "python"
-    }
-   },
-   "outputs": [],
-   "source": [
-    "# plot the H12 results here as well for convenience\n",
-    "ag3.plot_h12_gwss(\n",
-    "    contig=contig, \n",
-    "    window_size=window_size, \n",
-    "    analysis=phasing_analysis, \n",
-    "    sample_sets=sample_sets,\n",
-    "    sample_query=sample_query, \n",
-    "    min_cohort_size=min_cohort_size,\n",
-    "    max_cohort_size=max_cohort_size,\n",
-    ")"
-   ]
-  },
```

Comment:
Removed this cell as it was not behaving very well (sometimes not showing output) and also isn't really necessary because we plot the scans below if debug is True.